decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -126,10 +131,6 @@ def rubocop
         copy_file ".rubocop.yml", ".rubocop.yml"
       end
 
-      def ruby_version
```

Comment:
Why do you want to remove the `.ruby-version` addition?

Reply:
When i am playing with the app generator, I see that the .ruby-version is added twice ... Once from Rails app template generator, and once from Decidim. Prompting the user to take a decision whether to keep, override the ruby-version file.

Reply:
OK, but it might lead to another version that we would want them to use.
https://github.com/rails/rails/blob/05cb63abdaf6101e6c8fb43119e2c0d08e543c28/railties/lib/rails/generators/rails/app/templates/ruby-version.tt#L1

Some Decidim versions may not work with newer Ruby versions, so I think we would need to suggest them to use the preferred version through the `.ruby-version` file and leave it up for them if they want to try to run Decidim on a newer Ruby version.

So I would keep this method here. Shouldn't it override the method from the Rails generator?

https://github.com/rails/rails/blob/05cb63abdaf6101e6c8fb43119e2c0d08e543c28/railties/lib/rails/generators/rails/app/app_generator.rb#L53-L55

Reply:
Reverted the change. 
Rails is creating the `.ruby-version` file from `AppBuilder`, yet we extend `Rails::Generators::AppGenerator`.

Reply:
I don't think Rails is creating it because we are overriding that method.

I just tried re-creating the test app by changing the root `.ruby-version` to 3.1.2 and it correctly applied 3.1.1 to the test app without asking any questions. So I think it's working correctly.